docs/monitoring/rsv.md

```diff
@@ -0,0 +1,334 @@
+- **MyOSG and OIM Links.** RSV picks up resource information, WLCG interoperability information, etc., from a MyOSG resource group summary listing, which is in turn based on the [OSG Information Management (OIM) (topology) system](https://oim.grid.iu.edu) (Requires registration). Resource [maintenance scheduled on OIM](https://twiki.grid.iu.edu/twiki/bin/view/Operations/OIMMaintTool) are forwarded to WLCG SAM, if applicable.
+
+
+Before Starting
+---------------
+
+Before starting the installation process, consider the following points (consulting [the Reference section below](#reference) as needed):
+
+- **User IDs:** If they do not exist already, the installation will create the Linux user IDs `rsv` and `cndrcron`
+- **Service certificate:** The RSV service requires a service certificate (`/etc/grid-security/rsv/rsvcert.pem`) and matching key (`/etc/grid-security/rsv/rsvkey.pem`)
+- **Network ports:** To view results, port 80 must accept incoming requests; outbound connectivity to tested services must work, too
+- **Host choice:** Install RSV on your site CE unless you have specific reasons (e.g., performance) for installing on a separate host
+
+As with all OSG software installations, there are some one-time (per host) steps to prepare in advance:
+
+- Ensure the RSV host has [a supported operating system](../common/yum.md)
+- Obtain root access to the host
+- Prepare [the required Yum repositories](../common/yum.md)
+- Install [CA certificates](../security/ca-cert-auth.md)
+
+Installing RSV
+--------------
+
+An installation of RSV at a site consists of the RSV client software, the Apache web server, parts of HTCondor (for its cron-like scheduling capabilities), and various other small tools. To simplify installation, OSG provides a convenience RPM that installs all required software with a single command.
+
+1. Consider updating your local cache of Yum repository data and your existing RPM packages:
+
+        :::console
+        [root@client ~] # yum clean all --enablerepo=\*
+        [root@client ~] # yum update
+
+    !!! note
+        The `update` command will update **all** packages on your system.
+
+2. If you have installed HTCondor already but not by RPM, install a special empty RPM to make RSV happy:
+
+        :::console   
+        [root@client ~] # yum install empty-condor --enablerepo=osg-empty
+
+3. Install RSV and related software: 
+
+        :::console
+        [root@client ~] # yum install rsv  
+   
+Configuring RSV
+---------------
+ 
+After installation, there are some one-time configuration steps to tell RSV how to operate at your site.
+ 
+1. Edit `/etc/osg/config.d/30-rsv.ini` and follow the instructions in the file. There are detailed comments for each setting. In the simplest case — to monitor only your CE — set the `htcondor_ce_hosts` variable (or `gram_ce_hosts` for a GRAM CE) to the fully qualified hostname of your CE. For a sample `rsv.ini` file, see the complete installation output [below](#InstallDetails)
```

Comment:
Remove instructions for gram, and link to 'complete installation output below' (because it doesn't exist)